doc/dev/sharp_bits.rst

```diff
@@ -1049,12 +1049,41 @@ conditional statements, are also supported:
 >>> f(5)
 array([21., 21., 21., 21., 21.])
 
-However, capturing dynamic-shaped arrays within control-flow from outer scopes is currently not
-supported:
+By default, Catalyst for loops and while loops will automatically capture
+dynamically-shaped arrays from outside their scope for use within the loop,
+however loops cannot modify (i.e., return) array shapes and sizes:
```

Comment:
I think this structure is a bit off, if the parenthesis is read: "return array shapes and sizes".

Reply:
```suggestion
however loops cannot modify array shapes and sizes.
```